### PR TITLE
Admin interface theme configuration

### PR DESCRIPTION
**Pull request by lkiesow**
This patch adds the configuration to en-/disable themes in the admin interface.

This provides the configuration for https://github.com/opencast/opencast-admin-interface/pull/687

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://docs.opencast.org/develop/developer/#participate/development-process/#automatically-closing-issues-when-a-pr-is-merged) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: etc/org.opencastproject.organization-mh_default_org.cfg [hunk 1]
```diff
@@ -359,6 +359,11 @@ prop.admin.shortcut.add_media.previous_tab=shift+alt+left
 # Default: /editor-ui/index.html?id=$id
 #prop.admin.editor.url=/editor-ui/index.html?id=$id
 
+# Enable themes in the admin intterface.
+# Format: boolean
+# Default: false
+#prop.admin.themes.enabled=false
```
Comment (lkiesow) [inline etc/org.opencastproject.organization-mh_default_org.cfg]: The new default is `true` so we do not change anything in `r/15.x` but we want this to be disabled by default in 16. So we do that via this configuration:

```suggestion
# Default: true
prop.admin.themes.enabled=false
```

File: etc/org.opencastproject.organization-mh_default_org.cfg [hunk 2]
```diff
@@ -359,6 +359,11 @@ prop.admin.shortcut.add_media.previous_tab=shift+alt+left
 # Default: /editor-ui/index.html?id=$id
 #prop.admin.editor.url=/editor-ui/index.html?id=$id
 
+# Enable themes in the admin intterface.
```
Comment (Arnei) [inline etc/org.opencastproject.organization-mh_default_org.cfg]:
```suggestion
# Enable themes in the admin interface.
```

**Review by Arnei — CHANGES_REQUESTED** [review]
Makes sense to me. Found a typo:

**Comment by lkiesow** [comment]
Thanks for the review. Applied your suggested patch. Merging

**Pull request merged**